packages/cli/src/init/init.js

```diff
@@ -73,35 +73,36 @@ function generateProject(destinationRoot, newProjectName, options) {
     yarn.getYarnVersionIfAvailable() &&
     yarn.isGlobalCliUsingYarn(destinationRoot);
 
+  const packageManager = new PackageManager({
+    projectDir: destinationRoot,
+    forceNpm: options.npm,
+  });
+
   createProjectFromTemplate(
     destinationRoot,
     newProjectName,
     options.template,
     yarnVersion
   );
 
-  if (yarnVersion) {
-    logger.info('Adding React...');
-    execSync(`yarn add react@${reactVersion}`, { stdio: 'inherit' });
-  } else {
-    logger.info('Installing React...');
-    execSync(`npm install react@${reactVersion} --save --save-exact`, {
-      stdio: 'inherit',
-    });
-  }
-  if (!options['skip-jest']) {
-    const jestDeps = `jest babel-jest metro-react-native-babel-preset react-test-renderer@${reactVersion}`;
-    if (yarnVersion) {
-      logger.info('Adding Jest...');
-      execSync(`yarn add ${jestDeps} --dev --exact`, { stdio: 'inherit' });
-    } else {
-      logger.info('Installing Jest...');
-      execSync(`npm install ${jestDeps} --save-dev --save-exact`, {
-        stdio: 'inherit',
-      });
-    }
-    addJestToPackageJson(destinationRoot);
-  }
+  const DEPENDENCIES = [`react@${reactVersion}`];
+
+  const DEV_DEPENDENCIES = [
```

Comment:
Do we need these to be defined as separate variables? If yes, then definitely it doesn't need to be uppercase.

Reply:
We need to put in separate variables because they are installed differently. ALL_UPPER_CASE is a popular naming convention to mark constants.

Reply:
I mean why we don't use them inline, e.g.
`install([]` and `installDev([])`?

They are not really "constants" - at least `dependencies` is dynamic and depends on a `reactVersion`. You know, that way, every `const` variable should be UPPER_CASE :D

Reply:
Constant is not Javascript's `const`. But you are right about `dependencies` - it is dynamic.

Reply:
I know it's not - that's why I don't think we should uppercase variable declarations unless they are at the module level.